clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/AbstractGoogleServerGroupCachingAgent.java

```diff
@@ -667,6 +669,13 @@ private void setRegionConfig(GoogleServerGroup serverGroup, InstanceGroupManager
         .collect(toImmutableList());
   }
 
+  private static String getTargetShape(@Nullable DistributionPolicy distributionPolicy) {
```

Comment:
Since this method can return null should we annotate it with `@Nullable`?

Reply:
Done.